manifest.yml

```diff
@@ -60,6 +60,11 @@ schema:
    - "https://raw.githubusercontent.com/"
 
 versions:
+  "2.11.0":
+    description: Improve handling of unknown namespace index
+    changelog:
+      added:
+        - "Adds support for rebrowse triggers"
```

Comment:
This is exposed to users, so it should be more descriptive. Perhaps "Adds support for triggering rebrowse based on updates to namespacePublicationDate"